### PR TITLE
SYS-172 fix shardeum server patch files

### PR DESCRIPTION
**Pull request by chrypnotoad**
changes to the config file cause our patches to no longer apply. This PR updates the patches to work with the latest changes

Mainly the rotationEdgeToAvoid setting was changed to default to 0 and it was removed from the patches since it isn't going to be used

### CODE REVIEW

File: debug-300-nodes.patch [hunk 1]
```diff
@@ -32,13 +32,13 @@ index c14ab80..9ff5cd0 100644
...
-index 327a723..ff2c574 100644
+index bb9c1d6..4094dde 100644
 --- a/src/shardeum/shardeumFlags.ts
 +++ b/src/shardeum/shardeumFlags.ts
-@@ -131,7 +131,7 @@ export const ShardeumFlags: ShardeumFlags = {
+@@ -139,7 +139,7 @@ export const ShardeumFlags: ShardeumFlags = {
    DebugRestoreArchiveBatch: 2000,
    CheckNonce: true,
-   txNoncePreCheck: true,
+   txNoncePreCheck: false,
```
Comment (chrypnotoad) [inline debug-300-nodes.patch:41]: this is not part of the patch, but the default on the dev branch was set to false

**Review by muni-corn — APPROVED** [review]

File: debug-300-nodes.patch (hunk 1 above)
Comment (waisa3971) [reply debug-300-nodes.patch:41]: yeah
